### ISSUE TITLE
Input is too fast

### ISSUE THREAD

**Issue by mvforell** [cmdj13/WiiU-Keyboard#2]
When you press a character for longer than a tiny part of a second, it appears multiple times in the text.

**Comment by CreeperMario** [comment]
I have the exact opposite problem. If I hold the stylus over the key for more than ~0.1 seconds, the character is printed multiple times...

**Comment by mvforell** [comment]
Yeah, I've noticed that too when I was testing earlier today. I will look into it.

**Comment by mvforell** [comment]
Hm, I thought I could fix it using the WAIT constant, but the program doesn't seem to care. It's my first time coding back in C again for some time, maybe that's why I'm not seeing why that doesn't fix the issue.
